dojo_plugin/api/v1/score.py

```diff
@@ -0,0 +1,48 @@
+from flask import request
+from flask_restx import Namespace, Resource
+from CTFd.cache import cache
+from CTFd.models import Users
+
+from ...models import Dojos
+
+score_namespace = Namespace("score")
+
+def get_user_score(username):
+    # Alternative: hardcode max count and challenge id's for speed
+    # POSSIBLE BUG HERE - topics and courses dojos might result in doubling
```

Comment:
Changing challenge_ids to a set as per my change recommendation makes it clear this won't happen.